examples/todo.rs

```diff
@@ -30,16 +31,23 @@ use rustful::StatusCode;
 
 //Helper for setting a status code and then returning from a function
 macro_rules! or_abort {
-    ($e: expr, $response: expr, $status: expr) => (
+    ($e: expr, $response: ident, $error_message: ident) => (
```

Comment:
Expressions are more permissive than identifiers. Is there a particular reason for this change?